From a2150d038b3b42e28603cc14b14822625a9a634a Mon Sep 17 00:00:00 2001
From: w30004833 <wangzili6@huawei.com>
Date: Thu, 25 Mar 2021 10:07:47 +0800
Subject: [PATCH] [Huawei] put patch into repo

Offering: GaussDB Kernel
CVE: CVE_2020_28473.patch

More detail: fix bug for CVE_2020_28473

Signed-off-by: wangzili wangzili6@huawei.com
---
 bottle.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bottle.py b/bottle.py
index 910b747..26d8045 100644
--- a/bottle.py
+++ b/bottle.py
@@ -2579,7 +2579,7 @@ def parse_range_header(header, maxlen=0):
 
 def _parse_qsl(qs):
     r = []
-    for pair in qs.replace(';','&').split('&'):
+    for pair in qs.split('&'):
         if not pair: continue
         nv = pair.split('=', 1)
         if len(nv) != 2: nv.append('')
-- 
2.23.0

